go/cmd/vtctldclient/command/schema.go

```diff
@@ -301,6 +303,7 @@ func init() {
 	GetSchema.Flags().BoolVar(&getSchemaOptions.IncludeViews, "include-views", false, "Includes views in the output in addition to base tables.")
 	GetSchema.Flags().BoolVarP(&getSchemaOptions.TableNamesOnly, "table-names-only", "n", false, "Display only table names in the result.")
 	GetSchema.Flags().BoolVarP(&getSchemaOptions.TableSizesOnly, "table-sizes-only", "s", false, "Display only size information for matching tables. Ignored if --table-names-only is set.")
+	GetSchema.Flags().BoolVarP(&getSchemaOptions.TableSchemaOnly, "table-schema-only", "", false, "Display only table schemas in the result, skip columns and fields.")
```

Comment:
Maybe something like `skip including columns and fields metadata`? Now it reads as if there's maybe no column information at all, not even in the `CREATE` statements either.

Reply:
updated